File: 240128/씨 오 더블유 2/c-o-w-2.py
n=int(input())
a=list(input())

ans=0

for i in range(n-2):
    for j in range(i+1,n-1):
        for k in range(j+1,n):
            if a[i]=='C' and a[j]=='O' and a[k]=='W':
                ans+=1

print(ans)

File: 260106/씨 오 더블유 2/c-o-w-2.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');

const n = Number(input[0]);
const s = input[1];

// Please Write your code here.
let ans = 0;

for(let i=0; i<n; i++){
    for(let j=i+1; j<n; j++){
        for(let k=j+1; k<n; k++){
            if(s[i]==='C' && s[j]==='O' && s[k]==='W'){
                ans++;
            }
        }
    }
}

console.log(ans)